src/containers/timer/countdown/countdown-provider/countdown-provider.tsx

```diff
@@ -59,7 +71,7 @@ export default function CountdownProvider({ children }: Props) {
 
     timerRef.current = setInterval(() => {
       setLeftTime((prevTime) => {
-        if (prevTime <= 0) {
+        if (prevTime <= NO_TIME) {
           clearInterval(timerRef.current!);
           return 0;
         }
```

Comment:
타입스크립트에서 type assertion은 될 수 있으면 지양하는 것이 좋으니, if문에 조건을 추가해서 타입 가드를 하면 어떨까요?

```suggestion
        if (prevTime <= NO_TIME && timerRef.current) {
          clearInterval(timerRef.current);
          return 0;
        }
```